src/ccg2xml/ccg.ply

```diff
@@ -1043,10 +1038,10 @@ def install_feature(feature, lis, lineno):
 # Return XML to go in types.xml.
 def make_feature_types_xml():
     xml = []
-    for (x, featvals) in feature_to_values.iteritems():
```

Comment:
What is the reason for the call to `list()` in the new version? Can't we just iterate directly over `feature_to_values.items()`?

Also, the parentheses around `x, featvals` are not necessary.

Reply:
EIther it was a poor oversight, or this is how 2to3 handled it automatically. I assume the former and will update the code accordingly as well as search for other occurences like this one. Thanks a lot!